### DISCUSSION TITLE
Question About Configuring New System

### DISCUSSION THREAD

**Discussion by freecicero** [CyberShadow/aconfmgr#201] [Q&A]
I see one of the goals of aconfmgr is to "Quickly configure a new system, or restore an existing system according to a saved configuration."

Consistent with that, I thinking I can use aconfmgr to "save" my configuration on a current Arch machine, then on other machines install a base Arch install (just to get it booting and the drive partitioned correctly) and then take the aconfmgr folder from the first machine, copy it to the new machine, and then run "aconfmgr apply" to have it set up a full system with custom programs that is substantially the same (but not necessarily hardware identical) to the first machine.

Is this a correcting understanding of how aconfmgr can be used?

The reason I ask is:  Is there a quick and direct way to exclude the "hardware" settings so as not to overwrite the base grub and fstab and other settings on the new machine?

**Comment by CyberShadow** [comment]
Sorting out the configuration after the first run does require a bit of work, such as configuring ignore rules and sorting the configuration into things tied to the hardware of some particular machine versus things that apply to all machines.

See the sections [First run](https://github.com/CyberShadow/aconfmgr#first-run), [Ignoring some changes](https://github.com/CyberShadow/aconfmgr?tab=readme-ov-file#ignoring-some-changes), and [Managing multiple systems](https://github.com/CyberShadow/aconfmgr?tab=readme-ov-file#managing-multiple-systems) for details.
